src/constants.ts

```diff
@@ -0,0 +1,89 @@
+// @ts-ignore
```

Comment:
Why the `@ts-ignore`? Tip: You can add a comment after `// @ts-ignore`, e.g.

```
// @ts-ignore no type definitions for this library
```

Reply:
Oh, thanks for the tip! TIL